{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "C:\\Users\\29155\\Desktop\\pymarl-master\\pymarl-master\\src",
    "dependencies": [
      "numpy==1.24.1",
      "PyYAML==3.13",
      "sacred==0.8.4",
      "torch==2.1.0+cu118"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [],
    "sources": [
      [
        "main.py",
        "_sources\\main_328c828801d09c74b1f74b2c9c9c6102.py"
      ],
      [
        "run.py",
        "_sources\\run_3cdb9bb289eb4c837389e1bcbf573914.py"
      ],
      [
        "utils\\logging.py",
        "_sources\\logging_72bd8ccedd2c5bba193a80e4d7453dd5.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"C:\\Users\\29155\\anaconda3\\envs\\pymarl\\lib\\site-packages\\sacred\\config\\captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"C:\\Users\\29155\\Desktop\\pymarl-master\\pymarl-master\\src\\main.py\", line 42, in my_main\n    run(_run, config, _log)\n",
    "  File \"C:\\Users\\29155\\Desktop\\pymarl-master\\pymarl-master\\src\\run.py\", line 63, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"C:\\Users\\29155\\Desktop\\pymarl-master\\pymarl-master\\src\\run.py\", line 225, in run_sequential\n    logger.print_recent_stats()\n",
    "  File \"C:\\Users\\29155\\Desktop\\pymarl-master\\pymarl-master\\src\\utils\\logging.py\", line 48, in print_recent_stats\n    item = \"{:.4f}\".format(np.mean([x.items()[1] for x in self.stats[k][-window:]]))\n",
    "  File \"C:\\Users\\29155\\Desktop\\pymarl-master\\pymarl-master\\src\\utils\\logging.py\", line 48, in <listcomp>\n    item = \"{:.4f}\".format(np.mean([x.items()[1] for x in self.stats[k][-window:]]))\n",
    "AttributeError: 'tuple' object has no attribute 'items'\n"
  ],
  "heartbeat": "2023-10-25T16:23:45.929134",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz",
    "gpus": {
      "driver_version": "535.98",
      "gpus": [
        {
          "model": "NVIDIA GeForce GTX 1650",
          "persistence_mode": false,
          "total_memory": 4096
        }
      ]
    },
    "hostname": "DESKTOP-E551NIV",
    "os": [
      "Windows",
      "Windows-10-10.0.19045-SP0"
    ],
    "python_version": "3.8.18"
  },
  "meta": {
    "command": "my_main",
    "config_updates": {
      "env_args": {
        "map_name": "2s3z"
      }
    },
    "named_configs": [],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env_args.map_name=2s3z"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2023-10-25T16:19:47.408426",
  "status": "FAILED",
  "stop_time": "2023-10-25T16:23:45.934130"
}